src/main/java/sirius/biz/storage/VersionedFile.java

```diff
@@ -0,0 +1,95 @@
+package sirius.biz.storage;
+
+import sirius.biz.tenants.SQLTenantAware;
+import sirius.db.mixing.Mapping;
+import sirius.db.mixing.annotations.Length;
+import sirius.db.mixing.annotations.Lob;
+import sirius.db.mixing.annotations.NullAllowed;
+import sirius.db.mixing.annotations.Trim;
+
+import java.time.LocalDateTime;
+
+/**
+ * Entity holding meta information about a versioned file.
+ */
+public class VersionedFile extends SQLTenantAware {
```

Comment:
check via storage framework if active?

Reply:
done

Reply:
still add a @Framework(XXX) here so that the table isn't created if the framework isn't active.

also - shoudln't we move all this into a sub-package "versions" to not confuse it with VirtualObjectVersion(s)